### PR TITLE
Adding support for setting seed nodes for cassandra wrapper

### PR DESCRIPTION
**Pull request by Jan-M**
Add support for cassandra cluster entities containing list of seed nodes:

```
type: cassandra-cluster
name: my-cluster
seeds:
  - ip1
  - ip2
```

### CODE REVIEW

File: zmon_worker_monitor/builtins/plugins/cassandra_wrapper.py [hunk 1]
```diff
@@ -32,14 +32,30 @@ def create(self, factory_ctx):
...
 
 
 class CassandraWrapper(object):
     def __init__(self, node, keyspace, username=None, password=None, port=9042, connect_timeout=1, protocol_version=3):
         # for now using a single host / node should be seed nodes or at least available nodes
-        self.node = node
+
+        if node is not None and not isinstance(node, list):
+            if "," in node:
+                seeds = node.split(",")
```
Comment (hjacobs) [inline zmon_worker_monitor/builtins/plugins/cassandra_wrapper.py]: you can remove the if clause and always do split

File: zmon_worker_monitor/builtins/plugins/cassandra_wrapper.py [hunk 2]
```diff
@@ -32,14 +32,30 @@ def create(self, factory_ctx):
         :param factory_ctx: (dict) names available for Function instantiation
         :return: an object that implements a check function
         """
+
+        if factory_ctx.get('entity', {}).get('seeds') is not None:
```
Comment (hjacobs) [inline zmon_worker_monitor/builtins/plugins/cassandra_wrapper.py]: simply " if factory_ctx.get('entity', {}).get('seeds'):"

Comment (lmineiro) [inline zmon_worker_monitor/builtins/plugins/cassandra_wrapper.py]: Also, extract `factory_ctx.get('entity', {}).get('seeds')` to a variable. It's used again below

**Comment by Jan-M** [comment]
:+1:

**Comment by hjacobs** [comment]
:+1:

**Pull request merged**